packages/directory/tests/unit/controllers/directory-test.js

```diff
@@ -0,0 +1,8 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:directory', 'Unit | Controller | directory');
```

Comment:
Do we want to keep this test?

Reply:
Can you make these comments in #117. This isn't related to this PR.

Reply:
Also yes, we want to keep these. Once there is logic in place, we could just add the tests here.